Islandora-Devops/isle-dc
# 374-jp2-support Add manual selection strategy entries in Cantaloupe d…

To test:

1. Spin up a fresh ISLE-DC stack with make starter or make starter_dev
2. After logging in, go to /node/add/ and add a new Repository Item.
3. Choose 'Paged Content' for the model and save.
4. Click the 'Children' tab, and then "+batch upload children"
5. Choose 'Page' for Model, 'File' for Media Type and 'Original File' for Media Use, then click 'Next
6. Upload one or more JP2 files, and click 'Finish'
Click on the View tab and you should now see the Mirador viewer with the JP2 images displayed.

![image](https://github.com/Islandora-Devops/isle-dc/assets/82412/1c119947-c764-4db3-8dad-96bc14210da9)
